src/h2load.cc

```diff
@@ -1199,6 +1319,12 @@ Worker::~Worker() {
   ev_loop_destroy(loop);
 }
 
+void Worker::stop_all_clients() {
+  for(auto client: clients) {
+    client->~Client();
```

Comment:
The ownership is like: There can be a bunch of worker threads. Each worker owns a number of clients. Each client owns a number of sessions.
The worker was not calling all the clients' destructor explicitly. Probably, it was happening by calling implicit out-of-scope destructor. The workers were also not keeping any track or reference to the client object instances. I added that code to keep track and destroy/stop the clients, because we need this.
The other option was to inform all the clients through some variable in the (parent) worker. But that would not be more elegant, I thought.